be/src/vec/exec/vjdbc_connector.cpp

```diff
@@ -119,23 +119,13 @@ Status JdbcConnector::open(RuntimeState* state, bool read) {
     // Add a scoped cleanup jni reference object. This cleans up local refs made below.
     JniLocalFrame jni_frame;
     {
-        std::string local_location;
-        std::hash<std::string> hash_str;
-        auto* function_cache = UserFunctionCache::instance();
-        if (_conn_param.resource_name.empty()) {
-            // for jdbcExternalTable, _conn_param.resource_name == ""
-            // so, we use _conn_param.driver_path as key of jarpath
-            SCOPED_RAW_TIMER(&_jdbc_statistic._load_jar_timer);
-            RETURN_IF_ERROR(function_cache->get_jarpath(
-                    std::abs((int64_t)hash_str(_conn_param.driver_path)), _conn_param.driver_path,
-                    _conn_param.driver_checksum, &local_location));
+        std::string driver_path;
+
+        if (_conn_param.driver_path.find(":/") == std::string::npos) {
+            driver_path =  "file://" + config::jdbc_drivers_dir + "/" + _conn_param.driver_path;
```

Comment:
Driver path maybe an absolute path like: `/mnt/disk1/path/to/1.jar`, how to handle it?

Reply:
I added a path check on the FE side. The Driver URL is always sent by FE to BE, so BE is only responsible for processing it.